core/01_core.js

```diff
@@ -139,7 +148,15 @@
     const maybeError = opcallAsync(opsCache[opName], promiseId, arg1, arg2);
     // Handle sync error (e.g: error parsing args)
     if (maybeError) return unwrapOpResult(maybeError);
-    const p = PromisePrototypeThen(setPromise(promiseId), unwrapOpResult);
+    let p = PromisePrototypeThen(setPromise(promiseId), unwrapOpResult);
+    if (opCallTracingEnabled) {
+      const stack = StringPrototypeSlice(new Error().stack, 6);
```

Comment:
What's the significance of 6? Maybe add a comment or explanatory variable (probably using `"Error\n".length` is not appropriate for this code... I'm not sure about the overhead and we probably want to micro-optimize this).

Reply:
Added a comment.